### PR TITLE
Add `mhpl8r getrefr` command

### LINKED ISSUE #124: New command to grab hg38
MicroHapulator needs a new flag or subcommand to download and index the GRCh38 reference assembly.

https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/001/405/GCA_000001405.15_GRCh38/seqs_for_alignment_pipelines.ucsc_ids/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz

https://lh3.github.io/2017/11/13/which-human-reference-genome-to-use

### PR DESCRIPTION
**Pull request by standage**
New command line entry point for downloading a copy of the GRCh38 assembly suitable for whole-genome mapping. Closes #124.

**NOTE**: This was tested manually, but testing cannot be easily integrated into the test suite.

----------

- [x] Changes are clearly described above
- [x] Any relevant issue threads are referenced in the description
- ~~Any new features are tested (see [docs/DEVEL.md](docs/DEVEL.md) for details)~~
- [x] Substantial changes are documented in CHANGELOG.md (see https://keepachangelog.com/en/1.0.0/)

### CODE REVIEW

**Comment by standage** [comment]
Ready for review @danejo3!

File: microhapulator/cli/getrefr.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+from pathlib import Path
+from pkg_resources import resource_filename
+from subprocess import run
+import sys
+from tqdm import tqdm
+from urllib.request import urlretrieve
+
+
+class ProgressBar(tqdm):
+    """Stolen shamelessly from https://stackoverflow.com/a/53877507/459780."""
```
Comment (danejo3) [inline microhapulator/cli/getrefr.py:24]: 😂

**Comment by danejo3** [comment]
Looks good! Thanks! I was able to download GRCh38 just fine. It took 53 minutes to run the entire setup.

**Comment by standage** [comment]
Awesome. Thanks for confirming it works on your end! I'm hoping the terminal messages are sufficient warning to the user that indexing will take a long time.

**Pull request merged**